apps/knockdog/src/pages/kindergarten-detail-page/ui/KindergartenDetailPage.tsx

```
@@ -11,24 +11,29 @@ import { useParams } from 'next/navigation';
+import { useShare } from '@shared/lib/device';
 
 function KindergartenDetailPage() {
   const scrollableDivRef = useRef<HTMLDivElement>(null);
 
   const params = useParams();
   const rawId = (params as Record<string, string | string[] | undefined>)?.id;
   const id = Array.isArray(rawId) ? rawId[0] : rawId;
-  if (!id) throw new Error('Kindergarten ID is required');
+  if (!id) return null;
+
+  const { position } = useCurrentLocation();
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)